src/components/OverlayDrawer.tsx

```diff
@@ -5,14 +5,16 @@ import Image from 'next/image';
 import closeIcon from '@/../public/img/uswds/usa-icons/close.svg';
 
 export function OverlayDrawer({
```

Comment:
Would that require an additional loading indicator on the page, in case it takes a while for the drawer to open after clicking the button? (FWIW I don't expect loading to take nearly as long in prod as it does in dev)

Reply:
Hmm, yeah, if the delays are going to be longish on dev and/or variable in length, I think a loading state would make sense. I’d be happy to whip one up, if that’d be helpful.

Reply:
(but/and I could move that work over to #322 so you’re not blocked)

Reply:
Sounds great! On it.

Reply:
Just opened #487! Resolving this for now.